src/java.base/share/classes/java/lang/constant/MethodTypeDesc.java

```diff
@@ -188,6 +189,16 @@ default String displayDescriptor() {
                              returnType().displayName());
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * @apiNote {@linkplain MethodTypeDesc} can represent method type descriptors
+     * that are not representable by {@linkplain MethodType}, such as methods with
+     * more than 255 parameter slots, so attempts to resolve these may result in errors.
```

Comment:
Nah. Since such a MethodTypeDesc is never valid, such a MethodHandleDesc or DynamicConstantDesc should always fail to resolve for such a method handle can never be found. The only problem we need to be concerned about is that the failure in those Descs should be ReflectiveOperationException instead of some sort of UncheckedException (which I think MethodTypeDesc.resolveConstantDesc currently throws), but that belongs to another issue.

Reply:
agree, this `@apiNote` is specific for `MethodTypeDesc` only.